pages/travel.tsx

```diff
@@ -0,0 +1,16 @@
+import fs from "fs";
+import MiscLayout from "../components/miscLayout";
+
+import Travel from "../components/travel";
+
+interface CoCPageProps {
+  markdownContent: string;
+}
+
+export default function CoCPage({ markdownContent }: CoCPageProps) {
+  return (
+    <MiscLayout>
+      <Travel />
```

Comment:
Please add title of the page. This is being used for Google analytics.
Title: PyCon India 2023, Hyderabad | Travel
PR for reference which contains similar changes: https://github.com/pythonindia/inpycon2023/pull/150/files